app/services/builders/events/pull_request.rb

```diff
@@ -7,14 +7,17 @@ class PullRequest < Builders::Events::Base
 
       def build
         pull_request_data = @payload['pull_request']
+
         ::Events::PullRequest
-          .find_or_create_by!(github_id: pull_request_data['id']) do |pull_request|
+          .create_or_find_by(github_id: pull_request_data['id']) do |pull_request|
           ATTR_PAYLOAD_MAP.each do |key, value|
             pull_request.public_send("#{key}=", pull_request_data.fetch(value))
           end
           pull_request.branch = pull_request_data.dig('head', 'ref')
 
           assign_attrs(pull_request, pull_request_data)
+
+          pull_request.save!(validate: false)
```

Comment:
Usually, it's not a good idea to use `validate: false`. Why is it needed here?

Reply:
Otherwise it will fail because the validation of uniqueness `github_id`. Due to using `create_or_find_by` I'm sure that the record exists.